pennylane/template.py

```diff
@@ -209,10 +209,12 @@ def CVNeuralNetLayer(theta_1, phi_1, varphi_1, r, phi_r, theta_2, phi_2, varphi_
     Args:
         theta_1 (array[float]): length :math:`N(N-1)/2` array of transmittivity angles for first interferometer
         phi_1 (array[float]): length :math:`N(N-1)/2` array of phase angles for first interferometer
+        varphi_1 (array[float]): length :math:`N` array of rotation angles for first interferometer
```

Comment:
```suggestion
        varphi_1 (array[float]): length :math:`N` array of final rotation angles for first interferometer
```

Reply:
Added "final" to try to help distinguish the various angles better in people's heads